pages/documentation/getting-started-developers/phase-two.md

```diff
@@ -124,7 +124,10 @@ exports.compile = uswds.compile;
   ```
 
 ## Step 4: Create path settings and export compile functions
-Now, create your path settings using the following table:
+If you want to go to build or the dist directory and eliminate the USWDS subdirectory, create your path settings in the gulpfile. When you're initializing, the file pulls and copies elements from the node mod package of USWDS 3.0 and copies it into the project/source file while you are building in the dist file. 
```

Comment:
>If you want to go to build or the dist directory and eliminate the USWDS subdirectory

On re-read this makes sense. _As a user_ if I want to avoid including a directory in my project for USWDS SASS I would use USWDS Compile. The way you would do this is by **not** modifying `uswds.paths.src` settings.

```suggestion
If you want to use the default dist directory and eliminate the need for a project specific USWDS subdirectory, use the default `src` paths in the gulpfile. When you're initializing, the file pulls and copies elements from the node package of USWDS 3.0 and copies it into the project/source file (your defined `paths.dist` set
tings) while you are building. 
```

This will probably need more work, but I'm trying to say:

1. `paths.src` are default paths to USWDS  —  users **should not** need to modify these _unless_ they want to include USWDS dist files into their project.
2. `paths.dist` are project specific paths. Here they'll point to where they want specific assets to live. The most common use case is defining `paths.dist.theme` because this is where your SASS entry point and theme overrides will live.

Maybe we should remove the `paths.src` settings from the table and only show `dist` settings (project settings) with a link to the full table?